docs/pages/database-access/reference/configuration.mdx

```diff
@@ -85,14 +85,22 @@ proxy_service:
   # MySQL proxy is listening on a separate port and needs to be enabled
   # on the proxy server.
   mysql_listen_addr: "0.0.0.0:3036"
+  # Postgres proxy is listening address. If provided proxy will use a separate listener
+  # instead of multiplexing Postgres protocol on web_listener_addr.
+  postgres_listen_addr: "0.0.0.0:5432"
```

Comment:
Let's comment out these to indicate they're optional and by default web listener is used.
```suggestion
  # postgres_listen_addr: "0.0.0.0:5432"
```